test/test.py

```diff
@@ -156,3 +174,11 @@ def test_fc_lapack():
         assert 'dgesv test ok' in stdout
     else:
         pass
+
+
+def test_fc_lapack_static():
+    if sys.platform != 'win32':
+        stdout, stderr = configure_build_and_exe('fc_lapack', 'python setup.py --fc=gfortran --static --cmake-options="-DMATH_LIB_SEARCH_ORDER=ATLAS"')
```

Comment:
The choice of ATLAS is necessary for static linking: the native lapack.a and blas.a are crashing due to repeating object, see https://github.com/scisoft/autocmake/issues/51#issuecomment-127579836 .